# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../rhythmbox/libmtp.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./plugins/mtpdevice/rb-mtp-source.c
+++ ./plugins/mtpdevice/rb-mtp-source.c
@@ -681,7 +681,7 @@
 		trackmeta->filetype = mimetype_to_filetype (mimetype);
 	}
 
-	if (LIBMTP_Send_Track_From_File (device, filename, trackmeta, NULL, NULL, 0) != 0) {
+	if (LIBMTP_Send_Track_From_File (device, filename, trackmeta, NULL, NULL) != 0) {
 		LIBMTP_destroy_track_t (trackmeta);
 		rb_debug ("Tracktransfer failed\n");
 		return NULL;

